src/plugins/core/merge.ts

```diff
@@ -453,7 +454,7 @@ export class MergePlugin extends CorePlugin<MergeState> implements MergeState {
           const merge = this.getMerge(position);
           if (!merge || merge.id !== id) {
             this.history.update("mergeCellMap", sheetId, col, row, undefined);
-            this.dispatch("CLEAR_CELL", position);
+            this.dispatch("CLEAR_CELLS", { sheetId, target: [positionToZone(position)] });
```

Comment:
could we dispatch CLEAR_CELLS only once with the entire merge zone instead of position by position?

Reply:
the condition `if (!merge || merge.id !== id) ` prevents us from doing so.
I will let `CLEAR_CELL`